source/CommonJobs/CommonJobs.Application.EvalForm/Commands/RevertEvaluationStateCommand.cs

```diff
@@ -0,0 +1,94 @@
+﻿using CommonJobs.Application.EvalForm.Indexes;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using static CommonJobs.Application.EvalForm.Helper.RevertEvaluationActionsHelper;
```

Comment:
I think that `using static` is not compatible with previous VS versions, what do you think about to use a more conservative syntax?